lms/static/sass/course/_info.scss

```diff
@@ -1,11 +1,37 @@
+div.welcome-wrapper {
+  max-width: 1180px;
+  margin: 0 auto;
+
+  .welcome {
+    margin: $baseline;
+    h1 {
```

Comment:
it would be great to use .page-title here, similarly .page-subtitle below to avoid the element styling. 

(NIT: I would also use .home here for these new styles instead of .welcome )

I would also look at the teams header HTML to try and keep it closer to that if possible. That header uses more general title & subtitle classes, as shown below.

![image](https://cloud.githubusercontent.com/assets/2023680/11128336/30477678-8947-11e5-8be5-d15ac1aefe8f.png)